OSVR-Unity/Assets/OSVRUnity/src/ClientKit.cs

```diff
@@ -78,7 +85,20 @@ private void EnsureStarted()
             void Awake()
             {
                 DLLSearchPathFixer.fix();
-                DontDestroyOnLoad(gameObject);
+                //if an instance of this singleton does not exist, set the instance to this object and make it persist
+                if(_instance == null)
+                {
+                    _instance = this;
+                    DontDestroyOnLoad(this);
```

Comment:
This is harmless. Probably got that from here: http://unitypatterns.com/singletons/
Per the [docs](http://docs.unity3d.com/ScriptReference/Object.DontDestroyOnLoad.html), "If the object is a component or game object then its entire transform hierarchy will not be destroyed either."